src/openms/include/OpenMS/COMPARISON/SPECTRA/SpectrumAlignment.h

```diff
@@ -301,27 +303,15 @@ namespace OpenMS
         cerr << "(" << s1[it->first - 1].getPosition()[0] << " <-> " << s2[it->second - 1].getPosition()[0] << ") ("
              << it->first << "|" << it->second << ") (" << s1[it->first - 1].getIntensity() << "|" << s2[it->second - 1].getIntensity() << ")" << endl;
           }
-    #endif
-    #endif
-    }
-    else  // relative alignment (ppm tolerance)
-    {        
-      for (Size i = 0; i != s1.size(); ++i)
-      {
-        const double& theo_mz = s1[i].getMZ();
-        double max_dist_dalton = theo_mz * tolerance * 1e-6;
- 
-        // iterate over peaks in experimental spectrum in given fragment tolerance around theoretical peak
-        Size j = s2.findNearest(theo_mz);
-        double exp_mz = s2[j].getMZ();
-
-        // found peak match
-        if (std::abs(theo_mz - exp_mz) < max_dist_dalton)
-        {
-          alignment.push_back(std::make_pair(i, j));
-        }
+      #endif
+      #endif
```

Comment:
two endifs?

Reply:
yep. Pairs with 
```
#ifdef ALIGNMENT_DEBUG
#if 0
```
in the code. I did not touch that part though